### ISSUE TITLE
Header info for cache hits

### ISSUE THREAD

**Issue by gjn** [geoadmin/service-mapproxy#29]
It would be great to have information in the response header that say if the tile sent is coming from S3 backend cache or was calculated.

**Comment by gjn** [comment]
To get my feet wet, I can try to do this.

**Comment by procrastinatio** [comment]
My branch is https://github.com/procrastinatio/mapproxy/tree/s3

Every `cache`backend has a `is_cached` method, bht the trick is to get this info back to the service.

Should I wait before submiting my PR to `mapproxy`?

**Comment by gjn** [comment]
No. Please go ahead and submit your branch. It would be great if this would be integrated in s3 directly!

**Comment by gjn** [comment]
oops. I meant mapproxy, of course.
